### ISSUE TITLE
selected attribute not working when formcontrol is on select tag

### ISSUE THREAD

**Issue by stellasoft-will** [angular/angular#14116]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

when adding selected attribute to an option tag the formControl attribute on the select tag prevents the option from being selected even though the dom has a selected value.

**Expected behavior**

would like when selected value is set to have that option set in the select drop down

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

n/a
**Please tell us about your environment:**

webstrom, mac os, npm
* **Angular version:** 2.0.X

2.3.1
* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 all
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
typescript
* **Node (for AoT issues):** `node --version` =   
6.2.2

**Comment by DzmitryShylovich** [comment]
It's by design. You can find an explanation here https://github.com/angular/material2/issues/2250#issuecomment-274904991

**Comment by stellasoft-will** [comment]
ok so what should this actually look like:

`<select id="products" class="form-control new-select padding" formControlName="featured_product">
            <option>Please Select</option>
            <option *ngFor="let p of products" [ngValue]="p" [attr.selected]="p.ProductId == product.ProductId ? '': null" >{{p.ProductName}}</option>
          </select>`

**Comment by DzmitryShylovich** [comment]
https://github.com/angular/angular/pull/13349/files#diff-59d1e70711eb1e6eabdc76864164c9b4R2064

**Comment by metamaker** [comment]
nice, but why this PR is not still included?

**Comment by DzmitryShylovich** [comment]
¯\_(ツ)_/¯

**Comment by DzmitryShylovich** [comment]
@metamaker it was merged

**Comment by metamaker** [comment]
Thx for info. I will wait for release with this PR included and then will update my package.json.

**Comment by pkozlowski-opensource** [comment]
Fixed by f89d004c519fd1645b9f94ff40cd740ad3658360
